### ISSUE TITLE
add cloudy/clearsky flag to report options

### ISSUE THREAD

**Issue by wholmgren** [SolarArbiter/solarforecastarbiter_dashboard#225]
https://github.com/SolarArbiter/solarforecastarbiter-core/pull/382 improved support for the cloudy/clearsky flag in core. I believe we're now ready to support it in reports. Is that right @alorenzo175 ? If so, @lboeman we need to expose it here on the dashboard. I'm guessing/hoping that's a quick fix that can make it into beta5.

**Comment by lboeman** [comment]
Does this just entail adding another checkbox to the Quality flags options? If so, yes it will be a trivial update.

**Comment by alorenzo175** [comment]
Yes, to all. Can confirm it works when the CLEARSKY quality flag filter is set
https://dev-dashboard.solarforecastarbiter.org/reports/8e8472da-8579-11ea-88d5-0a580a8201c4

**Comment by alorenzo175** [comment]
hmm, CLEARSKY flag is only applied for GHI, so I'm not sure there should be a note about that or something

**Comment by lboeman** [comment]
Is it the case that if clearsky is selected, it is only applied to any GHI data in the report, or will an error be thrown if there isn't any GHI?

**Comment by wholmgren** [comment]
Maybe print it as `CLEARSKY (GHI only)`. 

We should also clarify the direction "Data flagged with any checked quality flags is excluded from analysis." I suggest "Observation data flagged with any of the checked quality flags is excluded from analysis. The corresponding forecast data is also excluded."

An error should not be thrown, though maybe a warning could be included.

**Comment by wholmgren** [comment]
I confirmed that there's no problem making a report with the `CLEARSKY` flag for either GHI or AC Power observations that do not already have the flag. The flag does show up in the data validation results table, but it has 0s. We could address removing it and printing a warning if it's not supported for that variable in another post-beta5 issue.

I forgot that `NIGHTTIME` also only applies to GHI (at least in prod - not sure if that was fixed in beta5), so we already have some experience in production with this kind of issue.
